src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import core.basesyntax.domain.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    private static List<Fruit> fruits = new ArrayList<>();
+
+    public static List<Fruit> getFruits() {
+        return fruits;
```

Comment:
Returning the original list 'fruits' directly allows external modification. Consider returning a copy of the list or an unmodifiable view.